packages/waku/src/lib/middleware/dev-server-impl.ts

```diff
@@ -112,6 +112,9 @@ const createMainViteServer = (
         rscHmrPlugin(),
         fsRouterTypegenPlugin(config),
       ],
+      resolve: {
+        conditions: ['import', 'module', 'default'],
```

Comment:
MainViteServer is used to bundle for browsers.
It should be in `ssr.resolve.conditions`.
Do we need to change RscViteServer?

Now I wonder if `build.ts` change is correct.

Reply:
I tested, `ssrLoadModule` won't use `browser` condition by default

Reply:
So, what's the original problem?
This MainViteServer it to for browsers, so `browser` condition should be appropriate.